server/channels/app/layer_generators/opentracing_layer.go.tmpl

```diff
@@ -8,6 +8,9 @@ package opentracing
 
 import (
 	"github.com/opentracing/opentracing-go/ext"
+
+	"github.com/mattermost/mattermost/server/public/shared/request"
```

Comment:
Is this right? I don't see the old request package here.

Reply:
If I don't put this, it uses the old path when I run `make app-layers`. I'm assuming this is like that because it uses the current upstream when filling in the imports.

Reply:
What do you mean by "current upstream"?

Reply:
Meaning that in order to fill in the "request" import that it uses it uses the current `request` package on `master` under the old location.